modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -14,7 +9,6 @@ import TextInput, {TextInputProps} from '@workday/canvas-kit-react-text-input';
 
 export interface ColorInputProps extends TextInputProps, GrowthBehavior {
   value: string;
-  error?: ErrorType;
```

Comment:
Nit: Technically this works because we spread `elemProps` onto the input component, but I still think we should expose `error` as a prop here just to be clear. It's the same as any other input specific props already included in `TextInputProps` that we also call out here.

This also involves pulling it out of `this.props` in `render` (which it looks like we had missed before):
```tsx
        const {

          //...

          error,
          ...elemProps
        } = this.props;

        //...

        <CustomHexInput
          inputRef={inputRef}
          onChange={this.handleChange}
          type="text"
          placeholder={placeholder}
          value={formattedValue}
          error={error}
          spellCheck={false}
          disabled={disabled}
          maxLength={7} // 7 to allow pasting with a hash
          {...elemProps}
        />
```

Reply:
I made this change

Reply:
It was moved down a few. It is still included now